korge/src/commonTest/kotlin/com/soywiz/korge/view/ViewsTest.kt

```diff
@@ -247,4 +251,98 @@ class ViewsTest : ViewsForTesting() {
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
+        countDownSprite.playAnimation(1)
+        assert(countDownSprite.bitmap.extract().contentEquals(digits[0]))
+    }
+
+    @Test
+    fun testSpriteAnimationPlay346Times() = viewsTest {
+        val countDownSpriteMap = resourcesVfs["countDown.png"].readBitmap().toBMP32()
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
+        countDownSprite.playAnimation(346)
+        assert(countDownSprite.bitmap.extract().contentEquals(digits[0]))
+    }
+
+    @Test
+    fun testSpriteAnimationPlay() = viewsTest {
+        val countDownSpriteMap = resourcesVfs["countDown.png"].readBitmap().toBMP32()
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
+        countDownSprite.playAnimation()
+        assert(countDownSprite.bitmap.extract().contentEquals(digits[0]))
+    }
+
+    @Test
+    fun testSpriteAnimationStartFrame() = viewsTest {
+        val countDownSpriteMap = resourcesVfs["countDown.png"].readBitmap().toBMP32()
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
+        countDownSprite.playAnimation(startFrame = 3, times = 10)
+        assert(countDownSprite.bitmap.extract().contentEquals(digits[3]))
+    }
+
+    @Test
+    fun testSpriteAnimationReversed() = viewsTest {
+        val countDownSpriteMap = resourcesVfs["countDown.png"].readBitmap().toBMP32()
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
+        countDownSprite.playAnimation(reversed = true, times = 1000)
+        assert(countDownSprite.bitmap.extract().contentEquals(digits[0]))
+    }
+
+    @Test
+    fun testSpriteAnimationReversedStartFrame() = viewsTest {
+        val countDownSpriteMap = resourcesVfs["countDown.png"].readBitmap().toBMP32()
+        val countDownAnimation = SpriteAnimation(countDownSpriteMap,24,36,0,0,10,1,0,0)
+        val countDownSprite = Sprite(countDownAnimation)
+
+        val digits = Array<Bitmap>(10){
+            countDownSpriteMap.extract(it*24, 0,24,36)
+        }
```

Comment:
If you move these tests to its own class, you can also move these locals into `lateinit var` properties, then use a `@Before` or a plain `suspend fun` to populate them, since seems to be the same for all the tests, right?